lib/tasks/us_census_api.rake

```diff
@@ -43,14 +43,18 @@ namespace :us_census_api do
 
   desc "Get the available variables from the US Census"
   task :vars, [] => [:setup, :environment] do
-    importer = GrdaWarehouse::UsCensusApi::Importer.new(years: @years, datasets: @datasets, state_code: @state_code, levels: @levels)
-    importer.bootstrap_variables!
+    @state_codes.each do |state_code|
```

Comment:
I think for the purposes of adding the variables, you don't need to do this twice.